src/uwtools/tests/test_cli.py

```diff
@@ -225,7 +225,16 @@ def test__dispatch_config_realize():
 
 
 def test__dispatch_config_realize_no_optional():
-    args, _ = cli._parse_args("uw config realize --values-file /foo.vals".split()[1:])
```

Comment:
Out of curiosity, why did you decide to revert back to this method? We had some pretty good arguments going for supportability and readability earlier today for using the `_parse_args` approach.

Reply:
From an offline conversation, "In a test function, if we have
args, _ = cli._parse_args("uw config translate".split()[1:])
We're throwing away the checks by assigning them to _. We keep the args. uwtools.cli.main() keeps both and then runs the check functions, including _check_file_vs_format() in this case. But, if our test either calls _parse_args() and discards the second return value, or if we just construct an argument dict ourselves, the "format is optional if filename is specified, required otherwise" check never runs."

Reply:
I think maybe we're all synced up due to Slack conversation, but the upshot, from my perspective is: If we use `_parse_args`, we really should use `STR` values instead of re-introducing literal strings like `config`, `realize`, `values-file`, etc., and that all gets so messy that I think just constructing the `args` `dict` manually is IMO cleaner. That also avoids the possibility that `_parse_args()` has a bug that makes should-be-unrelated tests fail.  I tried `_parse_args()` out on my branch and didn't like where it was going, so I dropped that and told @WeirAE that I thought he should, too.